bundle/regal/rules/bugs/not_equals_in_loop.rego

```diff
@@ -9,8 +9,7 @@ import future.keywords.in
 import data.regal.result
 
 report contains violation if {
-	some rule in input.rules
-	some expr in rule.body
+	expr := input.rules[_].body[_]
 
```

Comment:
Doesn't this lead to the same code once the compiler's stages ran over it...? 🤔

Reply:
Almost:

```rego
report[__local5__] {
	__local2__ = input.rules[__local1__];
	__local5__ = __local2__.body[__local4__]
}
```
vs
```rego
report[__local0__] {
	__local0__ = input.rules[_].body[_]
}
```
Whether the difference is meaningful? 🤷

Reply:
Also makes me curious about the whereabouts of `__local__3` :)